packages/components/pie-link/src/index.ts

```diff
@@ -49,27 +51,38 @@ export class PieLink extends LitElement implements LinkProps {
 
     render () {
         const {
-            variant, size, iconPlacement, href, target, rel, isBold, isStandalone,
+            tag,
+            variant,
+            size,
+            iconPlacement,
+            isBold,
+            isStandalone,
+            href,
+            target,
+            rel,
         } = this;
 
+        const element = unsafeStatic(tag);
+        const isButton = tag === 'button';
+
         return html`
-            <a
+            <${element}  
                 data-test-id="pie-link"
                 class="c-link"
+                tag="${tag}"
                 variant=${variant}
                 size=${size}
-                href=${ifDefined(href)}
-                target=${ifDefined(target)}
-                rel=${ifDefined(rel)}
                 ?isBold=${isBold}
-                ?isStandalone=${isStandalone}>
+                ?isStandalone=${isStandalone}
+                href=${!isButton && href ? href : nothing}
+                target=${!isButton && target ? target : nothing}
+                rel=${!rel && rel ? target : nothing}>
                     <div class="c-link-content">
-                      ${iconPlacement === 'leading' ? html`<slot name="icon"></slot>` : nothing}
-                      <slot></slot>
-                      ${iconPlacement === 'trailing' ? html`<slot name="icon"></slot>` : nothing}
+                        ${iconPlacement === 'leading' ? html`<slot name="icon"></slot>` : nothing}
```

Comment:
I don't think it's valid to have a `div` inside a `button` – may need to update it to be a `span` now?

Reply:
yep makes sense 👍